demonstrations/tutorial_qnn_multivariate_regression.py

```diff
@@ -0,0 +1,253 @@
+r"""
+Variational Quantum Circuit for Multidimensional Regression
+==========================================
+
+In this tutorial, we show how to use a variational quantum circuit to fit a simple multivariate function,
+:math:`f(x_1, x_2) = \frac{1}{2} \left( x_1^2 + x_2^2 \right)`. [#schuld]_
+shows that, under some conditions, there exist variational quantum circuits that are expressive enough to realize any possible set
+of Fourier coefficients. We will use a simple two-qubit parameterized quantum circuit to construct a partial Fourier series for fitting
+the target function.
+The main outline of the process is as follows:
+1. Build a circuit consisting of layers of alternating data-encoding and parameterized training blocks
+2. Optimize the expectation value of the circuit output against a target function that is the function that
+we want to fit
+3. Obtain a partial Fourier series for the target function; since the function is not periodic, this partial
+Fourier series will only approximate the function in the region we will use for training.
+4. Plot the optimized circuit expectation value against the exact function to compare the two.
+What is a quantum model?
```

Comment:
This list is not compiling properly for some reason. Not sure how to correct it.